cleanlab/classification.py

```diff
@@ -244,28 +250,28 @@ def fit(
         """This method implements the confident learning. It counts examples
         that are likely labeled correctly and incorrectly and uses their ratio
         to create a predicted confusion matrix.
-        This function fits the classifier (self.clf) to (X, s) accounting for
+        This function fits the classifier (self.clf) to (X, labels) accounting for
         the noise in both the positive and negative sets.
```

Comment:
After the first sentence, it's again a bit unclear that the latter docstring sentences are describing what this method does. Should docstring instead explain what this function does procedurally? (i.e. train `clf` on given data, run CL to find label issues, delete data with label issues, retrain `clf` on cleaned data to produce more robust version of the same classifier).

Reply:
Also unclear why "positive and negative sets" is mentioned in docstring. For some reason lots of the documentation seems to specifically mention binary (eg. "a binary vector") when this function works for multiclass classification.

Reply:
originally wrote this for binary. fixing and added more xplanation